keps/2724-topology-aware-scheduling/README.md

```diff
@@ -430,6 +431,14 @@ const (
   // at the highest topology level, then it gets admitted as distributed
   // among multiple topology domains.
   PodSetPreferredTopologyAnnotation = "kueue.x-k8s.io/podset-preferred-topology"
+
+  // PodSetUnconstrainedTopologyAnnotation indicates that a PodSet does not have any topology requirements.
+  // Kueue admits the PodSet if there's enough free capacity available.
+  // Recommended for PodSets that don't need low-latency or high-throughput pod-to-pod communication,
+  // but want to leverage TAS capabilities improve accuracy of admitting jobs
+  //
+  // +kubebuilder:validation:Type=boolean
+  PodSetUnconstrainedTopologyAnnotation = "kueue.x-k8s.io/podset-unconstrained-topology"
```

Comment:
Does this mean that all Pods need to have either annotations (`podset-required-topology`, `podset-preferred-topology` or `PodSetUnconstrainedTopologyAnnotation`)?

My previous question is what are differences no TAS annotation and `PodSetUnconstrainedTopologyAnnotation `.
It looks like both (no TAS annotation and PodSetUnconstrainedTopologyAnnotation) are same.

Reply:
In Kueue v0.10.0 PodSet needs to have one of annotation (required/preferred) in order to use TAS. The same goes with the unconstrained annotation, if a PodSet wants to leverage TAS it needs to have some annotation defines. So the difference between no annotation and the unconstrained annotation would be to whether or not use TAS. Using TAS even if there are no heavy pod-to-pod communication might be beneficial to mitigate resource fragmentation.

Please note that in Kueue v0.11.0 mimowo introduces a mechanism to implicitly default the annotation (to `preferred`: [lowest level]) if all RFs in a CQ has Topology defined. This KEP takes it into account, as we allow a user to change the implicit default (to `unconstrained`)

Reply:
/hold cancel
Thank you!